ui/panels-plugin/src/plugins/time-series-chart/VisualOptionsEditor.tsx

```diff
@@ -37,32 +38,37 @@ export function VisualOptionsEditor({ value, onChange }: VisualOptionsEditorProp
   };
 
   return (
-    <Stack spacing={1} alignItems="flex-start">
-      <Typography variant="overline" component="h4">
-        Visual
-      </Typography>
-      <Typography variant="h5">{VISUAL_CONFIG.point_radius.label}</Typography>
-      <Slider
-        data-testid={VISUAL_CONFIG.point_radius.testId}
-        defaultValue={DEFAULT_POINT_RADIUS}
-        valueLabelDisplay="auto"
-        step={0.5}
-        marks
-        min={0}
-        max={20}
-        onChange={handlePointRadiusChange}
+    <OptionsEditorGroup title="Visual">
```

Comment:
These are the ones I'm least sure about the look/feel because we don't have mocks for the sliders. I think it looks ok to start, and we may want to tweak it based on follow-ups with a designer.

Reply:
Yeah I can follow up on this to get feedback, I do really like the adjustments you made though at first glance